AnwesenheitsSimulation/README.md

```diff
@@ -1,4 +1,7 @@
 # Anwesenheits-Simulation
+
+_Die aktuelle Version dieses Moduls ist nicht mehr in SymconMisc verfügbar.\nDie aktuelle Version finden Sie in dem seit Version 5.1 verfügbaren Module Store.\nAlternativ können Sie es über das Module Control installieren: https://github.com/symcon/AnwesenheitsSimulation_
```

Comment:
\n funktioniert nicht in Markdown! Dies muss auch an diversen anderen Stellen angepasst werden